### ISSUE TITLE
Open customisable window rather than dialog window

### ISSUE THREAD

**Issue by petems** [petems/TimeToStretch#1]
Original idea: https://github.com/G07cha/pomodoro/issues/20

Basically, I'm trying to make an app that opens up an html file with a list of example stretches to do at given intervals.

I can't seem to find an example of a menubar app that opens a new Window when the app is not active. Is it possible?

I might open a bounty to get this done because I dunno how to Javascript 😢

**Comment by petems** [comment]
Note on the bounty: A pull-request to with the following is required - 
- When timer expires, instead of the `showMessageBox` happening, a new Window should open with a given in an HTML file

Current behaviour: 
<img width="420" alt="screenshot 2016-04-21 18 21 34" src="https://cloud.githubusercontent.com/assets/1064715/14718320/e7e62412-07ed-11e6-9cc8-0113e3cc0c33.png">

New behaviour:
<img width="484" alt="screenshot 2016-04-21 18 23 04" src="https://cloud.githubusercontent.com/assets/1064715/14718359/1f05ba84-07ee-11e6-9086-ff73c3c32787.png">

Bonus points if Window that appears is Full Screen/Maximised! 😄

**Comment by DayBr3ak** [comment]
Any news on this ?

**Comment by GarretKadeDupre** [comment]
@DayBr3ak Were you awarded your bounty?

**Comment by DayBr3ak** [comment]
@GarretKadeDupre No. The owner never answered. As far as I know Bountysource is just big waste of time.

**Comment by petems** [comment]
I clicked the button in Bountysource to start the reward process but it said it was waiting on you to accept. 

Bountysource appears to be down right now (https://twitter.com/Bountysource/status/1248045541184827392), but If you want to email me your paypal or cashapp or something I can send the money. 

Also, please dont baselessly call me a scammer because of something that happened 4 years ago with no follow up, I'd forgotten this repo even existed.

**Comment by petems** [comment]
@DayBr3ak Looks like bountysource is back up now, do you want to claim the reward? 

Go to https://www.bountysource.com/issues/33200201-open-customisable-window-rather-than-dialog-window, click solve issue, it should send it to you. 

They've got new ToS that if no-one claims it after two years, then they keep the money (which starts from July 2020) so you'll need to claim it now.

**Comment by DayBr3ak** [comment]
@petems Thanks for reaching. It's misunderstanding then, my bad. I'm terribly sorry for what I said.

It's seems I can't claim anything as long as this issue remains open.
